ignite3/src/main/java/site/ycsb/db/ignite3/IgniteJdbcClient.java

```diff
@@ -48,34 +48,29 @@ public void init() throws DBException {
     }
   }
 
+  /** {@inheritDoc} */
   @Override
   public Status read(String table, String key, Set<String> fields, Map<String, ByteIterator> result) {
     try {
-      String qry = prepareReadStatement(table);
+      PreparedStatement stmt = prepareReadStatement(CONN.get(), table);
```

Comment:
string interpolation in `prepare<*>Statement` is quite expensive and may affect result of measurements. I see that `IgniteAbstractClient.init()` is `Called once per DB instance; there is one DB instance per client thread.`. Does it make sense to do all the preparations during init phase?

Reply:
moved forming of prepared statement SQL strings to init()